src/main/java/nextstep/courses/domain/ImageSize.java

```diff
@@ -0,0 +1,28 @@
+package nextstep.courses.domain;
+
+import nextstep.courses.exception.ImageSizeException;
+
+public class ImageSize {
+    private static final int IMAGE_SIZE_MAX = 1024 * 1024;
+    public static final int IMAGE_SIZE_MIN = 0;
+
+    private final int imageSize;
+
+    public ImageSize(int imageSize) {
+        validImageSize(imageSize);
+        validSizeIsNegative(imageSize);
+        this.imageSize = imageSize;
+    }
+
+    private void validSizeIsNegative(int imageSize) {
+        if(imageSize <= IMAGE_SIZE_MIN) {
+            throw new IllegalArgumentException("이미지의 크기는 1바이트 이상이어야 합니다.");
+        }
```

Comment:
`IMAGE_SIZE_MIN` 값을 1로 설정하면 예외 메시지에도 해당 상수를 활용해볼 수 있을 것 같습니다.
```suggestion
        if (imageSize < IMAGE_SIZE_MIN) {
            throw new IllegalArgumentException("이미지의 크기는 " + IMAGE_SIZE_MIN + "바이트 이상이어야 합니다.");
        }
```